### ISSUE TITLE
No longer using old mesh

### ISSUE THREAD

**Issue by tonywright** [Turge08/print_area_bed_mesh#35]
Hi, I have noticed since the last update that the bed mesh is being done every time I call the macro regardless of the fact that the mesh size is exactly the same as previously. When looking at the macro there seems to be a whole section missing with this functionality. The "force" option is no longer available and I guess no longer needed if it is going to do the bed mesh every time. 
I am often finding myself having to restart a print so the nozzle and bed are up to temperature but I still have to wait for the bed mesh to be done.
Can this code be replaced into the current version or do I have to roll back to the old version that I didn't realise I had a problem with?

Thanks

Tony

**Comment by srbjessen** [comment]
I have also noticed mine running a new mesh every print. 
I have " BED_MESH_PROFILE LOAD=default " in my start gcode.
Even preloading the "old" mesh before starting, doesn't have any effect.
